src/tests/end-to-end/common/manifest-overide.ts

```diff
@@ -3,41 +3,30 @@
 import * as fs from 'fs';
 import * as util from 'util';
 
-export class ManifestOveride {
+export class ManifestInstance {
```

Comment:
nit: if we're renaming the class should we rename the file as well?

Reply:
> LGTM! Out of curiosity, how did you test this?

I tested this by running several iterations of the [e2e-reliability test](https://dev.azure.com/accessibility-insights/accessibility-insights-web/_build?definitionId=40&_a=summary) against this branch. In the first few commits I intentionally left browsers open to check whether any of the failure results were related to the manifest permission. Once that part seemed resolved I started closing browsers again and focused on the other bucket of flakiness. After creating the PR, ADO ran the same reliability check again and all 15 agents passed.